src/components/popups/BasicConfirmAlert.vue

```diff
@@ -0,0 +1,83 @@
+<script setup lang="ts">
+import { ref, onUnmounted } from 'vue';
+import QuestionIcon from '@/components/icons/Question.vue';
+defineProps({
+	text: {
+		type: String,
+		required: true,
+	},
+});
+
+const emit = defineEmits(['close', 'confirm']);
+const showInfoBubble = ref<boolean>(false);
+
+window.addEventListener(`click`, handleCloseClick, false);
```

Comment:
We don't need the event listener and its function anymore with the click.stop on the popup overlay